modules/luther/sphinx/datafile/__init__.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (C) 2011  Bradley N. Miller
```

Comment:
This should not be a part of this PR as the datafile directive is part of RunestoneComponents